vba_timeseries/models/value2pl/suuvid_obs_value2pl.m

```diff
@@ -0,0 +1,44 @@
+function  [ gx ] = suuvid_obs_nonu(Xt, phi, u, inG)
```

Comment:
The function name needs to match the filename. Here, this should be suuvid_obs_value2pl.